go.mod

```diff
@@ -63,38 +63,38 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.6
 	go.etcd.io/etcd/etcdutl/v3 v3.5.6
 	go.uber.org/zap v1.23.0
-	golang.org/x/crypto v0.1.0
-	golang.org/x/oauth2 v0.1.0
-	golang.org/x/tools v0.2.0
+	golang.org/x/crypto v0.3.0
+	golang.org/x/oauth2 v0.2.0
+	golang.org/x/tools v0.3.0
 	gomodules.xyz/jsonpatch/v2 v2.2.0
-	google.golang.org/api v0.101.0
-	google.golang.org/genproto v0.0.0-20221027153422-115e99e71e1c
-	google.golang.org/grpc v1.50.1
+	google.golang.org/api v0.103.0
+	google.golang.org/genproto v0.0.0-20221118155620-16455021b5e6
+	google.golang.org/grpc v1.51.0
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/yaml.v3 v3.0.1
-	helm.sh/helm/v3 v3.10.1
-	k8c.io/kubeone v1.5.2
+	helm.sh/helm/v3 v3.10.2
+	k8c.io/kubeone v1.5.3
 	k8c.io/operating-system-manager v1.1.1
-	k8s.io/api v0.25.3
-	k8s.io/apiextensions-apiserver v0.25.3
-	k8s.io/apimachinery v0.25.3
-	k8s.io/apiserver v0.25.3
+	k8s.io/api v0.25.4
```

Comment:
this (and others) will not be effective, you'll also need to update the `replace` sections below this.

Reply:
Good catch!